SECURITY.md

```diff
@@ -0,0 +1,45 @@
+# Security
+
+This document specifies the security process for the SwiftNIO project.
+
+## Disclosures
+
+### Private Disclosure Process
+
+The SwiftNIO core team asks that known and suspected vulnerabilities be
+privately and responsibly disclosed by emailing
+[cve@forums.swift.org](mailto:cve@forums.swift.org) with the [details usually
+included with bug reports][issue-template]. **Do not file a public issue.**
+
+#### When to report a vulnerability
+
+* You think you have discovered a potential security vulnerability in SwiftNIO
+  or any of the SwiftNIO projects.
+* You are unsure how a vulnerability affects SwiftNIO or any of the SwiftNIO
+  projects.
+
+#### What happens next?
+
+* A member of the team will acknowledge receipt of the report within 3
+  working days (United Kingdom). This may include a request for additional
+  information about reproducing the vulnerability.
+* We will inform the Swift Server Work Group ([SSWG][sswg]) of the vulnerability
+  within 10 days of the report.
```

Comment:
maybe add something like "as required by the sswg security guidelines" and add link to them